src/js/components/icons/Pulse.js

```diff
@@ -0,0 +1,32 @@
+// (C) Copyright 2014-2015 Hewlett Packard Enterprise Development LP
+
+import React, { Component, PropTypes } from 'react';
+import Add from './base/Add';
+import CSSClassnames from '../../utils/CSSClassnames';
+import classnames from 'classnames';
+
+const CLASS_ROOT = CSSClassnames.PULSE;
+
+export default class Pulse extends Component {
+  render () {
+    const classes = (this.props.className)
+      ? classnames(
```

Comment:
I believe classnames will handle that for us.

`const classes = classnames(CLASS_ROOT, this.props.className)` seems sufficient